ad713x-fmcz/ad713x.h

```diff
@@ -60,79 +60,79 @@
 #define AD713X_REG_PRODUCT_ID_LSB		0x04
 #define AD713X_REG_PRODUCT_ID_MSB		0x05
 #define AD713X_REG_CHIP_GRADE			0x06
```

Comment:
are these modifications reflected in a datasheet ? maybe add a link to it in the commit message